Polyjuice/Potions/Employee.cs

```diff
@@ -0,0 +1,18 @@
+using Polyjuice.Extension;
+using static Polyjuice.Potions.Data.DataReader;
+using GlobalCompany = Polyjuice.Potions.Company;
+
+namespace Polyjuice.Potions
+{
+    /// <summary>
+    /// An employee that works for an organization
+    /// </summary>
+    public static class Employee
+    {
+        public static string Username => Name.FirstName;
+        public static string FullName => $"{Username} {Name.LastName}";
+        public static string Enrollment => "########".Numerify();
+        public static string Company => GlobalCompany.CompanyName;
+        public static string Role => $"{Read("Job", "JobNouns").Rand()}";
```

Comment:
Use class [`Job`](https://github.com/yanjustino/polyjuice/blob/master/Polyjuice/Potions/Job.cs) to specify the Role attribute

```suggestion
        public static string Role => Job.Title;
```
If this suggestion is not applicable, you could implement a new attribute in `Job` class, such as:

```c#
   public static string SummaryTitle =>    $"{Read("Job", "JobNouns").Rand()} " ;
```

Reply:
@yanjustino Accepted the alternative suggestion